src/sidebar/store/modules/session.js

```diff
@@ -58,6 +64,27 @@ function isFeatureEnabled(state, feature) {
   return !!state.session.features[feature];
 }
 
+/**
+ * Should the tutorial panel be auto-displayed on client launch?
+ */
+function isTutorialAutoDisplayed(state) {
+  return (
+    state.viewer.isSidebar &&
```

Comment:
KNOWN ISSUE: `isSidebar` needs to be revised; this is not a "reliable" `isSidebar` indicator.

Reply:
If this function continues to live in the store, then it needs to depend only on state in the store, as otherwise its output could change without the store triggering a re-rendering of affected components. `viewer.isSidebar` could be initialized using the same utility function that is used outside the store elsewhere though.

If this function is moved to a helper that is used from within a component, then that requirement is dropped. It is then up to the component to either know that the value is static or update itself when it changes.

Reply:
> Also, I'm a bit confused by the function shouldShowSidebarTutorial inside session-utls which seems to be similar to this one? 

Part of this larger project is to untangle this. Much like there are multiple opinions about `isSidebar`...this needs to be sorted out. On my radar!